xdsl/irdl/declarative_assembly_format_parser.py

```diff
@@ -313,7 +315,9 @@ def parse_optional_variable(
 
         self.raise_error(
             "expected variable to refer to an operand, "
-            "attribute, region, result, or successor"
+            "attribute, region, result, or successor",
+            at_position=start_pos,
+            end_position=end_pos,
         )
 
     def parse_type_directive(self) -> FormatDirective:
```

Comment:
I would suggest submitting this through a separate PR, as this is indeed not in the experimental folder or taking this out of this PR.

Reply:
I think this is a change from main, if I click on 'View file' it takes me to the merge @superlopuh did from main, so I guess there's no need for a new PR?

Reply:
Can you merge latest main and make sure this diff disappears?